espn_api/basketball/player.py

```diff
@@ -46,4 +42,4 @@ def __repr__(self):
     
     def _stat_id_pretty(self, id: str):
         id_type = STAT_ID_MAP.get(id[:2])
-        return f'{id_type}_{id[2:]}' if id_type else id
\ No newline at end of file
+        return f'{id[2:]}_{id_type}' if id_type else id[2:]
```

Comment:
With the id change, you will also need to update lines 35-38 that use the id to set the players points field.

Reply:
Good call, done ✅